libr/core/cmd_seek.c

```diff
@@ -710,8 +738,16 @@ static int cmd_seek(void *data, const char *input) {
 		break;
 	}
 	case 'o': // "so"
-		cmd_seek_opcode (core, input + 1);
-		break;
+		switch (input[1]) {
+			case 'r':
+				if (input[2] == 't') {
+					cmd_sort (core, input);
+				}
+				break;
+			default:
+				cmd_seek_opcode (core, input + 1);
```

Comment:
probably better to just handle ' ' and \0 here, and show error on default

Reply:
Will do :)